web/app_options.js

```diff
@@ -260,7 +260,8 @@ if (
   defaultOptions.sandboxBundleSrc = {
     /** @type {string} */
     value:
-      typeof PDFJSDev === "undefined" || !PDFJSDev.test("PRODUCTION")
+      typeof PDFJSDev === "undefined" ||
+      PDFJSDev.test("!PRODUCTION && !INTEGRATIONTEST")
```

Comment:
Also, are these changes actually necessary since there *should* be no build-target setting `PRODUCTION = false` anyway?

Reply:
Good question, there was a `!PDFJSDev.test("PRODUCTION")`, is this value different of  `PDFJSDev.test("!PRODUCTION")` ?
And there's a similar use here:
https://github.com/mozilla/pdf.js/blob/master/web/app_options.js#L248

Reply:
> there was a `!PDFJSDev.test("PRODUCTION")`, is this value different of `PDFJSDev.test("!PRODUCTION")` ?

Those two formats are equivalent.